content/en/docs/devops-user-guide/how-to-use/devops-project-management.md

```diff
@@ -6,67 +6,54 @@ linkTitle: "DevOps Project Management"
-- You need to enable KubeSphere DevOps system.
+- You need to create a workspace and an account (`project-admin`). The account must be invited to the workspace with the role of `workspace-self-provisioner`. For more information, refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project/).
+- You need to enable the [KubeSphere DevOps system](../../../pluggable-components/devops/).
 
-## Create a DevOps 
+## Create a DevOps Project
 
-1. Sign in with project-admin, choose `DevOps Projects` tap, then click **Create** and select **Create a DevOps project**.
+1. Log in the console of KubeSphere as `project-admin`. Go to **DevOps Projects** and click **Create**.
 
-![Create a DevOps](/images/devops/create-devops-1.png)
+   ![devops-project-create](/images/docs/devops-user-guide/using-devops/devops-project-management/devops-project-create.jpg) 
 
-2. Fill in the basic information for this DevOps project.
+2. Provide the basic information for the DevOps project and click **OK**.
 
-- Name: A concise and clear name for this DevOps project, which is convenient for users to browse and search, e.g. `demo-devops`.
-- Description: A brief introduction to DevOps project.
+   ![create-devops](/images/docs/devops-user-guide/using-devops/devops-project-management/create-devops.jpg)
 
-![devops_create_project](/images/devops/create-devops-2.png)
+   - **Name**: A concise and clear name for this DevOps project, which is convenient for users to identify, such as `demo-devops`.
+   - **Alias**: The alias name of the DevOps project.
+   - **Description**: A brief introduction to the DevOps project.
+   - **Cluster Settings**: In the current version, a DevOps project cannot run across multiple clusters at the same time. If you have enabled [the multi-cluster feature](../../../multicluster-management/), you must select the cluster where your DevOps project runs.
 
-3. Then you will be able to view it has been created successfully. 
+3. A DevOps project will appear in the list below after created.
 
-![devops_create_project](/images/devops/create-devops-3.png)
+   ![devops-list](/images/docs/devops-user-guide/using-devops/devops-project-management/devops-list.jpg)
 
-## View the DevOps Project
+## View a DevOps Project
 
-4. Enter into `demo-devops` page, it allows DevOps project admin to create CI/CD Pipelines and Credentials, as well as project management which includes basic information, roles and members. 
+Click the DevOps project just created to go to its detail page. Tenants with different permissions are allowed to perform various tasks in a DevOps project, including creating CI/CD pipelines and credentials, and managing accounts and roles. 
 
-![devops_create_project](/images/devops/create-devops-4.png)
+![devops-detail-page](/images/docs/devops-user-guide/using-devops/devops-project-management/devops-detail-page.jpg)
```

Comment:
The screenshot seems incorrect.

Reply:
This is the page I am thinking about using here. It is where users will go to after they click the name of a DevOps project. Do you mean the name of the image should be changed?
![image](https://user-images.githubusercontent.com/65327072/100058210-ded3e200-2e63-11eb-9f53-d92d87fb8940.png)

Reply:
I mean the screenshot itself. This picture shows the page with no pipeline. But you actually just created one. And the section is to view the detail page. It seems confusing logically.

Reply:
Please note that no pipeline is created in this tutorial, whether in its original version or the one submitted in this pr. This tutorial talks about creating a DevOps project instead of creating pipelines.

Reply:
my bad